pkgs/development/python-modules/llvmlite/default.nix

```diff
@@ -28,34 +30,31 @@ buildPythonPackage rec {
     hash = "sha256-5QBSRDb28Bui9IOhGofj+c7Rk7J5fNv5nPksEPY/O5o=";
   };
 
-  nativeBuildInputs = [
-    llvm
-    setuptools
-  ];
+  build-system = [ setuptools ];
 
-  # Disable static linking
-  # https://github.com/numba/llvmlite/issues/93
-  postPatch = ''
-    substituteInPlace ffi/Makefile.linux --replace "-static-libstdc++" ""
+  buildInputs =
+    [ llvm ]
+    ++ lib.optionals withStaticLLVM [
+      zlib
+      libxml2.dev
+      ncurses
+    ];
 
-    substituteInPlace llvmlite/tests/test_binding.py --replace "test_linux" "nope"
+  postPatch = lib.optionalString withStaticLLVM ''
+    substituteInPlace ffi/build.py --replace "--system-libs --libs all" "--system-libs --libs --link-static all"
   '';
 
   # Set directory containing llvm-config binary
-  preConfigure = ''
-    export LLVM_CONFIG=${llvm.dev}/bin/llvm-config
-  '';
+  env.LLVM_CONFIG = "${llvm.dev}/bin/llvm-config";
 
   checkPhase = ''
     runHook preCheck
-    ${python.executable} runtests.py
```

Comment:
Why this change?

Reply:
Because `buildPythonPackage` will add `python` to `PATH` automatically.

Reply:
> Because `buildPythonPackage` will add `python` to `PATH` automatically.

Not the case for pypy and other python implementations?

```
nix-repl> pypy.executable
"pypy"
```